### PR TITLE
Add dialog components

### PR DESCRIPTION
**Pull request by FrogTheFrog**
@AAGaming00

### CODE REVIEW

**Comment by AAGaming00** [comment]
Is DialogButtonSecondary the same component as the old DialogButton?

**Comment by FrogTheFrog** [comment]
> Is DialogButtonSecondary the same component as the old DialogButton?

yes, but don't approve yet. I think I forgot to export Dialog

File: src/deck-components/Dialog.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import { CommonUIModule } from "decky-frontend-lib";
+import { CSSProperties, FC } from "react";
+
+export interface DialogCommonProps {
```
Comment (AAGaming00) [inline src/deck-components/Dialog.tsx:4]: HTMLAtrributes<HTMLDivElement>

**Review by AAGaming00 — DISMISSED** [review]

**Comment by AAGaming00** [comment]
Please resolve the conflicts when you can.

**Comment by FrogTheFrog** [comment]
WTF GITHUB

**Comment by FrogTheFrog** [comment]
WTF GITHUB

**Comment by AAGaming00** [comment]
I'll need to test this when I have a chance.

**Review by AAGaming00 — APPROVED** [review]

**Pull request merged**